sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 12.4.1 (2020-03-04)
+- Fixed a bug where EncryptedBlobClient.upload methods would not encrypt users data. Now the method throws an UnsupportedOperationException.
```

Comment:
Can you change this to call out the specific method that wasn't encrypting? It's a bad look in our changelog to say "the encryption client never encrypted" and that's not quite true.

Reply:
done